audit/format.go

```diff
@@ -557,3 +578,16 @@ func doElideListResponseData(data map[string]interface{}) {
 		}
 	}
 }
+
+// getMountPluginType returns the plugin type based the mount accessor of a logical.Request.
+func getMountPluginType(req *logical.Request) string {
+	if req.MountAccessor == "" {
+		return ""
+	}
+
+	if strings.HasPrefix(req.MountAccessor, fmt.Sprintf("%s_", consts.PluginTypeCredential.String())) {
```

Comment:
This doesn't seem quite watertight to me. What if someone (admittedly no one sensible) creates a kv mount like this?

`vault secrets enable -path=auth_kv kv`

Reply:
Thank you!